
[Executed at: Wed Mar 14 1:54:07 PDT 2018]

Traceback (most recent call last):
  File "/home/ccc_v1_c79523__48809/asn24778_45/asn24782_4/asnlib/submission.py", line 8, in <module>
    from q6_work import find_path
  File "./q6_work.py", line 13
    else:
        ^
IndentationError: unindent does not match any outer indentation level
Command exited with non-zero status 1
